docs/clustering/multi-zone-setup.rst

```diff
@@ -5,124 +5,188 @@ CrateDB multi-zone setup
 ========================
 
 If possible, we recommend running all CrateDB nodes of a cluster inside the
-same physical space (e.g. data center) to minimize network latency and maximize
+same physical space (i.e. data center) to minimize network latency and maximize
```

Comment:
It doesn't have to be considered a 'data center', does it? You can run it locally, no?

Reply:
@matthijskrul agreed

also, minor: there should be a comma after the second period